src/game/InputConstants.h

```diff
@@ -1,4 +1,66 @@
 #pragma once
+#ifndef _INPUTCONSTANTS_H
+#define _INPUTCONSTANTS_H
+#ifndef INI_MAX_LINE
+#define INI_MAX_LINE 200
+#endif
+
+#include <stdio.h>
+#include <string>
+#include <stdlib.h>
+#include <fstream>
+#include <map>
+
+static std::map<int,std::string> actions;
```

Comment:
It would be best if this maps int to int (mapping key enum to event enum). Loading configs from file only happens once in a while, but handling user input can happen multiple times a second, so it's preferable to perform the event name -> event value conversion while loading the configurations.